docs/data/getting-started/overview/overview.md

```diff
@@ -0,0 +1,37 @@
+---
+title: MUI X - Overview
+---
+
+# MUI X - Overview
+
+<p class="description">MUI X is a collection of advanced React UI components for complex use cases.</p>
+
+## Introduction
+
+MUI X is a collection of advanced UI components, including our [Data Grid](/x/react-data-grid) and [Date and Time Pickers](/x/react-date-pickers/getting-started).
+
+These components are significantly more complex than anything found in the MUI Core libraries, and feature advanced functionality for data-heavy applications and a wide range of other use cases.
+
+> MUI X is **open core**—base components are MIT-licensed, while more advanced features require a Pro or Premium commercial license.
+> See [Licensing](/x/getting-started/licensing/) for details.
+>
+> Thoughout the documentation, Pro- and Premium-only features are denoted with the [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/) and [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-premium/) icons, respectively.
+
+## Advantages of MUI X
+
+- **Ship faster:** Our team has invested thousands of hours into these components so you don't have to. Get up and running in a fraction of the time it would take to build from scratch.
```

Comment:
```suggestion
- **Ship faster:** Our team has invested thousands of hours in these components so you don't have to. Get up and running in a fraction of the time it would take to build from scratch.
```